pkg/ifuzz/arm64/gen/gen.go

```diff
@@ -0,0 +1,134 @@
+// Copyright 2024 syzkaller project authors. All rights reserved.
+// Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
+
+// gen generates instruction tables (ifuzz_types/insns.go) from ARM64 JSON.
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"strconv"
+	"strings"
+
+	"github.com/google/syzkaller/pkg/ifuzz/arm64"
+	"github.com/google/syzkaller/pkg/serializer"
+	"github.com/google/syzkaller/pkg/tool"
+)
+
+// nolint: gocyclo, gocognit, funlen, dupl
```

Comment:
Does it really produce all of these warnings? It does not look too long/compelx.